### ISSUE TITLE
None of the variables come through in an Lambda Invoke

### ISSUE THREAD

**Issue by iamtomcat** [apex/up#726]
## Prerequisites

* [x] I am running the latest version. (`up upgrade`)
* [x] I searched to see if the issue already exists.
* [x] I inspected the verbose debug output with the `-v, --verbose` flag.
* [x] Are you an Up Pro subscriber?

## Description

When I invoke a function through the lambda interface it seems to call it just fine, but none of the variables passed in make it through with the request. Just trying to see if I missed something in the setup.

## Steps to Reproduce

```

**Comment by tj** [comment]
hmm yeah might be a little tricky, you'd have to craft the right object to match what API Gateway would invoke Lambda with, then it shoulddddd be ok. https://github.com/apex/up/blob/master/internal/proxy/request.go can maybe provide a bit of insight there

**Comment by iamtomcat** [comment]
Cool I'll take a look into it.

**Comment by iamtomcat** [comment]
Just wondering where would I build this object. Trying to figure out the flow. Seems like currently js shim -> main (which I'm assuming is the proxy) -> server (my go code).

Maybe I'm just missing something here.

**Comment by tj** [comment]
Your plan was to skip API Gateway? If I understand that right then you'd have to invoke with that object, althoughhhh I'm actually not sure if it's even possible to popular the context object manually, that would be in AWS' / API Gateway's control I'd think.

**Comment by iamtomcat** [comment]
Yeah it's kind of a one off thing I'm trying to do. Just writing a lamda authenticator for auth0.

https://github.com/PayNoMind/lambda-authenticator

I got it working by switching the handler to be main and use go runtime directly. Also still grabs all the logs and etc, which is nice.

I've got another project that I'm working on getting up working with that is going through the api gateway.

**Comment by iamtomcat** [comment]
Ah So I think I figured out what the issue is, the type events that come through aren't always Api Gateway requests. So I have to look into extending the proxy to support other types.
